### PR TITLE
Added fields to registration actions

### LINKED ISSUE #83: Добавление новых поле в блок регистрационных действий
Вопрос                     | Ответ
-------------------------- | :---:
Наименование группы филдов | `registration_actions`

### Добавляемые филды

Ниже перечислены филды, которые необходимо добавить в файл спецификации согласно правилам, описанным в [readme-файле][readme]:

```json
{
        "path": "registration_actions.items[].owner.phone_number",
        "description": "Регистрационные действия: Предполагаемый номер телефона владельца",
        "types": [
            "string"
        ],
        "fillable_by": [
            "base",
            "base.ext"
        ]
    }
```

[readme]:https://github.com/avtocod/specs/blob/master/README.md

### PR DESCRIPTION
**Pull request by Yuri364**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes

## Description

Added new fields:

- Field with path `registration_actions.items[].owner.phone_number`

Fixes #83 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code
- [x] I have made changes in [CHANGELOG.md](https://github.com/avtocod/specs/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## v3.11.0
+
+### Added
+
+- Field with path `registration_actions.items[].owner.phone_number` [#83]
+- Field with path `registration_actions.items[].geo.postal_code` [#83]
```
Comment (tarampampam) [inline CHANGELOG.md]: `registration_actions.items[].geo.postal_code` is not defined in `fields/default/fields_list.json`. So, we really should add this entry in changelog?

File: fields/default/fields_list.json [hunk 2]
```diff
@@ -775,6 +775,17 @@
...
         ]
     },
+    {
+        "path": "registration_actions.items[].owner.phone_number",
+        "description": "Регистрационные действия: Предполагаемый номер телефона владельца",
+        "types": [
+            "string"
+        ],
+        "fillable_by": [
+            "base",
```
Comment (tarampampam) [inline fields/default/fields_list.json]: Source named `base` really provides this field?

File: reports/default/json-schema.json [hunk 3]
```diff
@@ -1369,6 +1369,21 @@
...
+                                            "null"
+                                        ],
+                                        "examples": [
+                                            null,
+                                            "9123456789"
+                                        ],
+                                        "fillable_by": [
+                                            "base",
+                                            "base.ext"
+                                        ]
```
Comment (tarampampam) [inline reports/default/json-schema.json]: Any string length limits?

File: reports/default/json-schema.json [hunk 4]
```diff
@@ -1394,7 +1409,30 @@
...
+                                                        ],
+                                                        "examples": [
+                                                            null,
+                                                            "141207"
+                                                        ],
+                                                        "fillable_by": [
+                                                            "base",
+                                                            "base.ext"
+                                                        ]
+                                                    }
```
Comment (tarampampam) [inline reports/default/json-schema.json]: String length limits, pattern rule, anything else?

File: fields/default/fields_list.json (hunk 2 above)
Comment (Yuri364) [reply fields/default/fields_list.json]: Yes

File: reports/default/json-schema.json (hunk 3 above)
Comment (Yuri364) [reply reports/default/json-schema.json]: Added definition

File: reports/default/json-schema.json (hunk 4 above)
Comment (Yuri364) [reply reports/default/json-schema.json]: Added pattern

**Pull request merged**